simulator/simulator.go

```diff
@@ -240,11 +240,14 @@ func (s *Service) RoundTrip(ctx context.Context, request, response soap.HasFault
 
 	// Every request has a "This" field.
 	this := req.Elem().FieldByName("This")
+	// Copy request body
```

Comment:
@hickeng rather than worry about detail such as shallow-copy of the esx.EthernetCard template, added this so each VM gets a full/deep copy of the create/reconfig spec.